pytorch_to_returnn/torch/nn/utils/rnn.py

```diff
@@ -0,0 +1,32 @@
+from ..modules.rnn import PackedSequence, BatchSizesDummy
+from ..modules.shape import FlattenBatch
+from ...tensor import Tensor
+from ....naming import Naming
+
+
+def pack_padded_sequence(input: Tensor, lengths=None, batch_first=False, enforce_sorted=True) -> PackedSequence:
+  return PackedSequence(FlattenBatch(batch_major=batch_first)(input))
+
+
+def pad_packed_sequence(sequence: PackedSequence, batch_first=False, padding_value=0.0, total_length=None):
+  assert padding_value == 0.0, "not implemented yet"
+  assert total_length is None, "not implemented yet"
+
+  return sequence.get_batched_tensor(), sequence.batch_sizes or BatchSizesDummy()
+
+
+def pack_sequence(sequences, enforce_sorted=True):
+  assert enforce_sorted, "not implemented yet"
+  if isinstance(sequences, Tensor):
```

Comment:
Could it ever be not a `Tensor` here?

Reply:
Ah, I guess yes, otherwise there is no real difference to `pack_padded_sequence`, right?

Reply:
Yes, actually it should not be a `Tensor` but a `list[Tensor]`, see https://pytorch.org/docs/stable/generated/torch.nn.utils.rnn.pack_sequence.html

But it also works with a batched tensor.